.travis.yml

```diff
@@ -10,10 +11,16 @@ services:
 install:
   - pip install molecule docker
 
-script:
+before_script:
   - molecule --version
```

Comment:
This is good to know, but not a test. :D

Reply:
LOL...No love on this one eh? :)